src/Sentry/Span.cs

```diff
@@ -199,8 +180,8 @@ public static Span FromJson(Transaction parentTransaction, JsonElement json)
                 Description = description,
                 Status = status,
                 IsSampled = isSampled,
-                _tags = tags,
-                _data = data
+                _tags = tags!,
+                _extra = data!
```

Comment:
I'm not a big fun of `!`, maybe check if its null and set a new Dic if null? just for the sake of nul safety

Reply:
I agree.
we can't fully guarantee that the map is not nullable in JSON, but we're responsible for both serialization and deserialization parts here, so we can reasonably guarantee it.
in general though, it would be better to add a proper check, but it's on our other list of improvements: #631

Reply:
We had a few experiences with deserializing something that we expect to be in a certain format only to find out it's not.

I agree with @marandaneto here and even though not the nicest thing, ideally we'd:

`tags ?? new Dictionary<>()` to avoid a null reference exception if we're accessing `tags` from  a deserialized instance.

Reply:
@bruno-garcia it's not the `tags` that is null here. The error is raised because the second generic argument in dictionary is nullable here.

Reply:
Oh, I see now. Not clear at all from just `!`. Lost chance to have a nicer syntax

`tags<>!` :P

Reply:
Indeed, that would have been much better :P